pkgs/tools/video/harvid/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub, ffmpeg, libjpeg, libpng, pkg-config }:
+stdenv.mkDerivation rec {
+  pname = "harvid";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "x42";
+    repo = "harvid";
+    rev = "v${version}";
+    sha256 = "0l1plfsfh2ixhlzg3hqqvjj42z7g422718a9kgbh7b4p882n71x7";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ ffmpeg libjpeg libpng ];
+
+  makeFlags = [ "DESTDIR=$(out)" ''libdir="/lib"'' ];
```

Comment:
This is another case where I wrote it this way originally, but `nixfmt` changed it *to* this.

You mention that it doesn't really matter what nixfmt thinks, but from my understanding the whole point of using a tool with some sort of standard formatting like that is so that we don't have to waste time bike-shedding over styling like we have here and [here](https://github.com/NixOS/nixpkgs/pull/115121#discussion_r589720640).

It seems that I was mistaken in assuming that `nixfmt` (a package in this repository) was some sort of standard tool in the same way that rustfmt is to rust. That said, you seem to have some auto-formatting check or a specific styling in mind. Can you point me to a tool you're using or a document I can refer to so that we can avoid the back and forth on formatting nits?

Reply:
> This is another case where I wrote it this way originally, but nixfmt changed it to this.

Yet another reason to use nixpkgs-fmt.

> Can you point me to a tool you're using or a document I can refer to so that we can avoid the back and forth on formatting nits?

nixpkgs-fmt does not enforce such formatting changes but it will also not revert the current way.

Double single quotes is reserved for multiline strings and single line strings should use double quotes.
```
''
...
''

"..."
```

Reply:
> Yet another reason to use nixpkgs-fmt.

After a week, this is the first comment where you've mentioned `nixpkgs-fmt` in review of this PR at all. I did not realise the tool even existed until yesterday.

I personally do not care which tool is used at all - just as long as there is some standard one so that I do not have to guess which maintainer I'm going to get and decipher their personal styling preferences by trawling through the recent PRs they've landed.

> nixpkgs-fmt does not enforce such formatting changes but it will also not revert the current way.

I highly recommend that if you have no direct reference in the standard documentation or standard automated tool that users have access to, that you be more lenient in PRs on these finer styling nits. Otherwise, if it means enough to you, I recommend updating the necessary tools, contributing docs and/or manual accordingly.